libr/anal/esil.c

```diff
@@ -424,6 +430,18 @@ R_API bool r_anal_esil_pushnum(RAnalEsil *esil, ut64 num) {
 	return r_anal_esil_push (esil, str);
 }
 
+R_API bool r_anal_esil_pushbig(RAnalEsil *esil, ut8 *num, ut32 size) {
+	RNumBig *big_num = r_big_new();
+	memcpy(big_num, num, size % R_BIG_ARRAY_SIZE);
+	return r_anal_esil_push (esil, r_big_to_hexstr(big_num));
+}
+
+// these numbers are jacked
+R_API RNumBig* r_anal_esil_get_big(RAnalEsil *esil, const char *str, RNumBig *big_num) {
+	r_big_from_hexstr(big_num, str);
```

Comment:
pleaes, follow the codingstyle (space before (, braces opening at right, ..), `} else {` in one line..